CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+
+If you run into an issue with the project:
+- Create a new [GitHub issue](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/issues/new?&template=bug_report.md) with our bug report template.
+	- Alternatively you can submit your request through the [FullStory Help Center](https://help.fullstory.com/hc/en-us/requests/new) with details about the error you're running into.
+- Provide the information you collected in the previous section. 
+- For faster remediation you should isolate the problem and create a reduced test case or provide a minimal reproducible example.
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps.
+- If the team is able to reproduce the issue, it will be handled accordingly by our support team. We will do our best to maintain an open line of communication as we work on your request.
+
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion such as a completely new feature or minor improvements to existing functionality. Following these guidelines will help maintainers to understand your suggestion.
+
+- Make sure that you are using the latest version.
+- Find out if the functionality is already covered.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to convince the maintainers of the merits of this feature. We want features that will be useful to the majority of users.
+
+#### Creating a Feature Request
+
+Enhancements are suggested by creating a new [GitHub issue](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/issues/new?&template=feature_request.md) with our feature request template.
+
+- Use a descriptive title for the issue to identify the suggestion.
+- Provide a step-by-step description of the suggested enhancement, include screenshots and animated GIFs if necessary.
+- Describe the current behavior and explain which behavior you expected to see instead. 
+- Describe alternatives to your solution and why the alternatives do not work for you.
+- Explain why this enhancement would be useful.
+
+## Pull Request Process
+
+All pull requests should include a clear list of your changes (read more about [pull requests](http://help.github.com/pull-requests/)). Any pull request that makes functional changes to the library should also include unit tests. We can always use more test coverage. 
+
+Always write a clear log message for your commits. One-line messages are fine for small changes, but bigger changes should look like this:
+
+```
+$ git commit -m "A brief summary of the commit"
+> 
+> A paragraph describing what changed and its impact."
```

Comment:
Looks like there's a trailing `"` in the extended paragraph commit message.

Reply:
fixed should read like so. the user needs to enter a few new lines to add a description.

```
$ git commit -m "A brief summary of the commit
> 
> A paragraph describing what changed and its impact."
```